test/setup.js

```diff
@@ -75,8 +78,13 @@ function endTest (app, t, err) {
 function screenshotCreateOrCompare (app, t, name) {
   const ssDir = path.join(__dirname, 'screenshots', process.platform)
   const ssPath = path.join(ssDir, name + '.png')
-  fs.ensureFileSync(ssPath)
```

Comment:
This `fs-extra` method is easy to replicate with a try-catch around `fs.readFileSync`